source/katana/index.md

```diff
@@ -0,0 +1,105 @@
+            <h1>The modern and lightweight CardDAV and CalDAV server.</h1>
+            <a class="install" href="{{ site.url }}/katana/install">
+                <i class="fa fa-download"></i>
+                Install
+            </a>
+        </div>
+        <div class="box-list">
+        <ul class="sheet-features">
+            <li>
+                <i class="fa fa-refresh fa-fw"></i>
+                Contact, calendar & task syncing via CardDAV/CalDAV.
+            </li>
+            <li>
+                <i class="fa fa-magic fa-fw"></i>
+                Easy installation
+            </li>
+            <li>
+                <i class="fa fa-sitemap fa-fw"></i>
+                Modern architecture
+            </li>
+            <li>
+                <i class="fa fa-users fa-fw"></i>
+                Built-In user management.
+            </li>
+            <li>
+                <a href="{{ site.url }}/katana/install">
+                    <i class="fa fa-file-text fa-fw"></i>
+                    Awesome documentation
+                </a>
+            </li>
+        </ul>
+        </div>
+    </div>
+    <div class="box-shadow"></div>
+</section>
+
+<section class="box box--sandbottom text--center">
+    <h1>Manage your own server</h1>
+    <figure class="image-cutbottom">
+        <img src="{{ site.url }}/img/screen-katana-admin.png" alt="Log in screen for sabre/katana">
+    </figure>
+    <div class="box-shadow"></div>
+</section>
+
+<section class="box box--ice text--center">
+    <h1>Powered by</h1>
+    <a class="logo" href="{{ site.url }}/">
+        <img src="{{ site.url }}/img/logo.png" alt="sabre/dav">
+        <br>
+        sabre<span>/dav</span>
+    </a>
+        <h2>The leading CardDAV, CalDAV and WebDAV framework.</h2>
+        <a href="http://sabre.io" class="btn">sabre.io</a>
+    <div class="panel panel--t panel--trustedby">
+        <p>Proven technology that is trusted by:</p>
+        <a href="https://www.atmail.com" title="sabre/dav is trusted by atmail.">
+            <img src="{{ site.url }}/img/trusted/atmail.png" alt="atmail works with sabre/dav">
+        </a>
+        <a href="http://tech.blog.box.com/2014/10/in-search-of-an-open-source-webdav-solution/" title="sabre/dav is trusted by Box.">
+            <img src="{{ site.url }}/img/trusted/box.png" alt="Box works with sabre/dav">
+        </a>
+        <a href="https://fruux.com" title="sabre/dav is developed by fruux.">
+            <img src="{{ site.url }}/img/trusted/fruux.png" alt="Our consumer product fruux is powered by sabre/dav">
+        </a>
+        <a href="http://owncloud.org" title="sabre/dav is trusted by ownCloud.">
+            <img src="{{ site.url }}/img/trusted/owncloud.png" alt="ownCloud works with sabre/dav">
+        </a>
+    </div>
+</section>
+
+<section class="box box--sand box--katanasupport text--center">
+    <h1>Enterprise support</h1>
+    <img src="{{ site.url }}/img/trusted/fruux.png" alt="Logo fruux">
```

Comment:
We've already address this in the support page as well. It's still supposed to be an open source website.